src/test/java/uk/co/ramp/policy/isolation/SingleCaseIsolationPolicyTest.java

```diff
@@ -141,17 +137,27 @@
 
   @Before
   public void setUp() {
-    when(flatZeroPercent.getDistributionValue()).thenReturn(0);
-    when(flatZeroPercent.max()).thenReturn(0D);
-    when(flatHundredPercent.getDistributionValue()).thenReturn(100);
-    when(flatHundredPercent.max()).thenReturn(100D);
-    when(flatOneDay.getDistributionValue()).thenReturn(1);
-    when(flatOneDay.max()).thenReturn(1D);
-    when(flatThreeDays.getDistributionValue()).thenReturn(3);
-    when(flatThreeDays.max()).thenReturn(3D);
+
+    //    flatZeroPercent = createMockBoundedDistribution(0,0);
+    //    flatHundredPercent = createMockBoundedDistribution(100,100);
+    //    flatOneDay = createMockBoundedDistribution(1,1);
+    //    flatThreeDays = createMockBoundedDistribution(3,3);
+    //    thresholdLinearBoundedDistribution = TestUtils.createMockBoundedDistribution(50,100);
+    //    when(flatZeroPercent.getDistributionValue()).thenReturn(0);
+    //    when(flatZeroPercent.max()).thenReturn(0D);
+    //    when(flatHundredPercent.getDistributionValue()).thenReturn(100);
+    //    when(flatHundredPercent.max()).thenReturn(100D);
+    //    when(flatOneDay.getDistributionValue()).thenReturn(1);
+    //    when(flatOneDay.max()).thenReturn(1D);
+    //    when(flatThreeDays.getDistributionValue()).thenReturn(3);
+    //    when(flatThreeDays.max()).thenReturn(3D);
+    thresholdLinearBoundedDistribution = createMockBoundedDistribution(50, 100);
     when(thresholdLinearBoundedDistribution.getDistributionValue())
         .thenAnswer(returnsElementsOf(List.of(5, 15, 25, 35, 45, 55, 65, 75, 85, 95)));
     when(thresholdLinearBoundedDistribution.max()).thenReturn(100D);
```

Comment:
I believe line 154, `when(thresholdLinearBoundedDistribution.max()).thenReturn(100D);` is redundant.